pkg/sdn/registry/netnamespace/rest.go

```diff
@@ -0,0 +1,163 @@
+package netnamespace
+
+/*
+Only needed for backward compatibility with older clients (openshift <= 3.2 release)
```

Comment:
Use Origin version numbers here, also do not use comments in the `/*` style, use `//`.